models/helpers.ts

```diff
@@ -230,18 +270,52 @@ export function getTransactionStatusColumn(): ColumnConfig {
   };
 }
 
+export function getLeadStatusColumn(): ColumnConfig {
+  return {
+    label: t`Status`,
+    fieldname: 'status',
+    fieldtype: 'Select',
+    render(doc) {
+      const status = getLeadStatus(doc) as LeadStatus;
+      const color = statusColor[status] ?? 'gray';
+      const label = getStatusTextOfLead(status);
+
+      return {
+        template: `<Badge class="text-xs" color="${color}">${label}</Badge>`,
+      };
+    },
+  };
+}
+
+// export async function createCustomer(fyo: Fyo, doc: Doc) {
+//   const partyDoc = fyo.doc.getNewDoc(ModelNameEnum.Party, {
+//     ...doc.getValidDict(),
+//     fromLead: doc.name,
+//     phone: doc.mobile as string,
+//     role: 'Customer',
+//   });
+//   return partyDoc;
+// }
+
```

Comment:
why the below lines are commented ? . if it is unnecessary code, you can delete it.